apiserver/params/params.go

```diff
@@ -375,6 +375,9 @@ type UpdateApplicationUnitArgs struct {
 type UpdateApplicationUnits struct {
 	ApplicationTag string                  `json:"application-tag"`
 	Scale          *int                    `json:"scale,omitempty"`
+	Status         string                  `json:"status,omitempty"`
```

Comment:
do we wanna have a status struct field instead here?

Reply:
Yeah, I was being consistent with other places, but will change.